modules/build/src/main/scala/scala/build/errors/ScalaNativeBuildError.scala

```diff
@@ -0,0 +1,5 @@
+package scala.build.errors
+
+final class ScalaNativeBuildError(
+  caughtError: String
+) extends BuildException(s"Error compiling with Scala Native:\n${caughtError}")
```

Comment:
Let's not pass an argument here, if we don't really use it yet.
```suggestion
final class ScalaNativeBuildError extends BuildException("Error compiling with Scala Native.")
```